email/spool.rst

```diff
@@ -141,3 +141,16 @@ You can also set the time limit in seconds:
 Of course you will not want to run this manually in reality. Instead, the
 console command should be triggered by a cron job or scheduled task and run
 at a regular interval.
+
+.. caution::
+
+    When you use the filesystem for spooling, messages are serialized with
+    the message class name. If the service is lazy loaded this class name
+    change on every cache clear. So if you send a mail, then you clear the
+    cache, the message will not be unserialisable.
```

Comment:
`unserialisable` -> `unserializable` (because we use American English instead of British English)